test/Sets/Tetrahedron.jl

```diff
@@ -10,11 +11,18 @@ for N in [Float64, Float32, Rational{Int}]
     @test Tmat == T
 
     # suport vector
-    @test σ(ones(3), T) == N[0, 1, 1/sqrt(2)]
+    @test σ(ones(3), T) == N[0, 1, 1 / sqrt(2)]
 
     # membership
     @test zeros(N, 3) ∈ T
 
     # is_polyhedral
     @test is_polyhedral(T)
+
+    # LazySets#3303
+    T = Tetrahedron([N[0.0, 14.999499879684222, 0.0],
+                     N[0.0, 0.0, 0.0],
+                     N[15.468646044437914, 0.0, 0.0],
+                     N[15.468646044437914, 0.0, 3.1024194077510114]])
```

Comment:
Here is a simpler case.

```suggestion
    T = Tetrahedron([N[0, 0, 0], N[0, 1, 0], N[1, 0, 0], N[1, 0, 1]])
```